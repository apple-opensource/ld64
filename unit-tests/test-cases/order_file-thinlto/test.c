__attribute__((visibility("hidden")))
int foo() asm("_foo.llvm.ABCDEFG1");
int foo() { return 0; }

void api() {
  foo();
}
