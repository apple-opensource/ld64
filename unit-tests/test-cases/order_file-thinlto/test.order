_bar
_foo
_api3
_api
_api2
