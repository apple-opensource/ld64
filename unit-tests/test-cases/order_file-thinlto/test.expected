_bar
_foo
_api3
_api
_api2
_foo
